docs/source/devguide/build_system.rst

```diff
@@ -512,7 +512,7 @@ Variable                   Description                                      Allo
 ``COMPILER``               Specifies the compiler                           gnu, intel, clang, bluegene      gnu
```

Comment:
Yes, we actually did. I'm not going to claim that I'm confident it will work anymore but it did back then.